src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/stash/StashApiClient.java

```diff
@@ -385,13 +392,7 @@ public String call() throws StashApiException, IOException {
                             + httpResponse.getStatusLine().getReasonPhrase());
                   }
 
-                  InputStream responseBodyAsStream = httpResponse.getEntity().getContent();
-                  StringWriter stringWriter = new StringWriter();
-                  IOUtils.copy(responseBodyAsStream, stringWriter, "UTF-8");
-                  response = stringWriter.toString();
-                  logger.log(Level.FINEST, "API Request Response: " + response);
```

Comment:
I am not sure how much this logger was used in practice, but can see its usefulness in troubleshooting. Wonder if it should be revived in `responseAsString()`?..

Reply:
I see this could be useful, though you should be able to easily dump selected requests on the fronting proxy, load balancer etc. as well. 
I would generally prefer to avoid maintaining functionalities that can be realized elsewhere

Reply:
actually what about

>  The whole response was logged twice in postRequest() at the FINEST level.
   Log it only once.